internal/bazel_integration_test/BUILD.bazel

```diff
@@ -21,8 +21,6 @@ nodejs_binary(
     name = "test_runner",
     data = ["@npm//tmp"],
     entry_point = ":test_runner.js",
```

Comment:
the comment here was about reducing memory - I still think it may be useful to leave the flag so users can get out of trouble if source_map_support is really the wrong thing for their program

Reply:
I was reducing memory at the time to get bazel-in-bazel working but I don't think it was material in the effort as its not a large library. The bazel and JVM memory flags are far more significant so I should just remove that comment.

As far as leaving it in. It doesn't hurt anything to do so. In the API review we decided to remove it which is where this PR came from.